hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## 0.2.17
+
+Released 2019-07-10
+
+### New features
+
+- (hail#6349) Added `compression` parameter to `export_block_matrices`, which can
+  be `'gz'` or `'bgz'`.
+- (hail#6405) When a matrix table has string column-keys, `matrixtable.show` uses
+  the column key as the column name.
+- (hail#6345) Added an improved scan implementation, which reduces the memory
+  load on master.
+- (hail#6462) Added `export_bgen` method.
+
```

Comment:
(hail#6473) Improved performance of `hl.agg.array_sum` by about 50%.
(hail#6498) Added method `hl.lambda_gc` to calculate the genomic control inflation factor.
(hail#6456) Dramatically improved performance of pipelines containing long chains of calls to `Table.annotate`, or `MatrixTable` equivalents.
(hail#6506) Improved the performance of the generated code for the `Table.annotate(**thing)` pattern.